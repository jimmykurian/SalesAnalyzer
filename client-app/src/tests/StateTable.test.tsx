import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow } from "enzyme";
import StateTable from "../feature/stateTable";

configure({ adapter: new Adapter() });
test("renders the component", () => {
  const props = { columns: ([] = []), rows: ([] = []) };
  const component = shallow(<StateTable {...props} />);
  expect(component).toMatchSnapshot();
});
