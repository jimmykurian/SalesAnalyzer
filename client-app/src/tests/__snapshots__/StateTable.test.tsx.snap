// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<div
  style={
    Object {
      "background": "#FFFFFF",
      "height": 400,
      "maxWidth": 1200,
      "width": "100%",
    }
  }
>
  <Memo([object Object])
    columns={Array []}
    pageSize={15}
    rows={Array []}
  />
</div>
`;
